internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
```

Comment:
Consider adding validation rules for the struct fields, particularly for `PullRequest` and `Repository`, to ensure the integrity of data being processed. For example, check that IDs are always positive integers, strings are not empty, etc. This will help catch potential logical errors when dealing with incoming data.